x-pack/solutions/security/plugins/security_solution/public/flyout/entity_details/user_right/index.tsx

```diff
@@ -182,6 +188,7 @@ export const UserPanel = ({ contextID, scopeId, userName, isPreviewMode }: UserP
               isPreviewMode={isPreviewMode}
               isLinkEnabled={isLinkEnabled}
             />
+            {assetInventoryEnabled && <UserPanelFooter userName={userName} />}
```

Comment:
Are we suppose to show this footer on Preview mode as well ?
we hide it for Host panel, why do we show it on User panel ?